### ISSUE TITLE
Sample app is not recognizing end of speech

### ISSUE THREAD

**Issue by nospam2k** [willblaschko/AlexaAndroid#82]
I'm not sure if I'm misunderstanding something. In the sample app, I press the button to begin recording speech but when I stop talking, it's still continues to listen.

**Comment by syedtalha** [comment]
You have to implement end of speech detection on your own. Or if you were able to get the downchannel service working, just subscribe to the events, and when you get a directive of type "StopCapture", stop listening in your app.

**Comment by nospam2k** [comment]
Yes, that was exactly what I discovered. I setup a subscribed event.
